Engines/Wine/Plugins/Font smoothing/script.js

```diff
@@ -0,0 +1,35 @@
+include("engines.wine.engine.object");
+
+/**
+ * force the Font smothing
+ * @param {string} mode ("RGB", "BGR" or "Gray Scale")
+ * @returns {Wine} Wine object
+ */
+Wine.prototype.FontSmoothing = function (mode) {
+    var FontSmoothingType;
+    var FontSmoothingOrientation;
```

Comment:
variable names should be written in Lowercase, while classes are normally written in Uppercase